flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/sql/JoinITCase.scala

```diff
@@ -372,9 +372,163 @@ class JoinITCase(
     val table = CollectionDataSets.getSmall3TupleDataSet(env).toTable(tEnv).as('a1, 'a2, 'a3)
     tEnv.registerTable("A", table)
 
-    val sqlQuery1 = "SELECT * FROM A CROSS JOIN (SELECT count(*) FROM A HAVING count(*) < 0)"
-    val result = tEnv.sql(sqlQuery1).count()
+    val sqlQuery1 = "SELECT * FROM A CROSS JOIN " +
+      "(SELECT count(*) FROM A HAVING count(*) < 0)"
+    val result = tEnv.sql(sqlQuery1)
+    val expected =Seq(
+      "2,2,Hello,null",
+      "1,1,Hi,null",
+      "3,2,Hello world,null").mkString("\n")
```

Comment:
this test was change bacause now we join with null record. look [testSingleRowLeftOuterJoinWithNullRecords](https://github.com/apache/flink/pull/3033/files#diff-102e5d9e330260c0acf5e4e54ff3bdceR478)